framework/test/bundles/libLQ/resources/manifest.json

```diff
@@ -1,8 +1,23 @@
 {
-  "bundle.symbolic_name" : "TestBundleLQ",
-  "bundle.testproperty" : "YES",
-  "bundle.activator" : true,
-  "hosed" : "1",
-  "hosedd" : "2",
-  "hose" : "3"
+    "bundle.symbolic_name" : "TestBundleLQ",
+    "bundle.testproperty" : "YES",
+    "bundle.nestedproperty" : {
+        "foo" : "bar"
+    },
+    "bundle.activator" : true,
+    "hosed" : "1",
+    "hosedd" : "2",
+    "hose" : "3",
+    "a" : {
```

Comment:
would be useful to add test cases that test the order of search for a scenario like:

```
"a.b" : "foo",
"a" : { 
         "b" : "bar"
}
```

Reply:
Done.